### PR TITLE
Log requested compression of response and compression of request. 

### PR DESCRIPTION
**Pull request by antban**
Add values from request headers to nakadi.access.log event type
- accept_encoding
- content_encoding

These fields may be used to analyze if clients are making use of request compression for publishing and response compression for consumption.

### CODE REVIEW

**Comment by adyach** [comment]
there are could be tests requiring fixing as well

**Comment by antban** [comment]
:+1:

File: core-common/src/test/java/org/zalando/nakadi/repository/kafka/KafkaRecordDeserializerTest.java [hunk 1]
```diff
@@ -105,6 +107,8 @@ private ObjectNode expectedJsonEvent() {
                 .put("app_hashed", "hashed-app")
                 .put("status_code", 201)
                 .put("response_time_ms", 10)
+                .put("accept_encoding", "-")
```
Comment (adyach) [inline core-common/src/test/java/org/zalando/nakadi/repository/kafka/KafkaRecordDeserializerTest.java:110]: why do we need `-`, it eats space. we can have null in there.

Comment (antban) [reply core-common/src/test/java/org/zalando/nakadi/repository/kafka/KafkaRecordDeserializerTest.java:110]: In this particular test - it is a test. 
Overall - I am trading stability of schema over disk usage.

Comment (adyach) [reply core-common/src/test/java/org/zalando/nakadi/repository/kafka/KafkaRecordDeserializerTest.java:110]: 👍  makes sense to keep the current values as it is!

**Comment by adyach** [comment]
👍

**Comment by antban** [comment]
:+1:

**Pull request merged**